clients/java/signalr/src/main/java/com/microsoft/signalr/HubMessageType.java

```diff
@@ -10,7 +10,8 @@
     STREAM_INVOCATION(4),
     CANCEL_INVOCATION(5),
     PING(6),
-    CLOSE(7);
+    CLOSE(7),
+    INVOCATIONBINDINGFAILURE(-10);
```

Comment:
If anything a generic error message which wraps the actual error which we could reuse for other things that could come up.

Reply:
I'd be OK with `INVOCATIONPARSINGFAILURE` I guess, but it still needs to be clearly an "invocation-related" error. .NET uses `InvocationBindingFailure` so I think I prefer consistency here (though that's just for our benefit rather than any customer-facing consistency)